app/assets/javascripts/chf_ga_events.js

```diff
@@ -0,0 +1,14 @@
+// Sufia/hyrax JS for sending event tracking to Google Analytics is a bit inflexible.
+// https://github.com/samvera/hyrax/blob/1e504c200fd9c39120f514ac33cd42cd843de9fa/app/assets/javascripts/hyrax/ga_events.js
```

Comment:
Yep, it's not an override, it's separate JS. The one built into Hyrax (currently) looks for elements with a certain id, and assumes they have a certain GA event/label, and looks up the GA value from the 'label' tag. 

This one looks for elements with a data-analytics-event attribute, and gets all GA properties from data-* attributes. 

So, yeah, they'll apply to different elements, and both work, coexisting.  I thought it was worth establishing our own convention, cause the one built into sufia/hyrax is very inflexible. 

Theoretically this could be upstreamed to hyrax, but then you have to worry about breaking existing code, and I just don't really want to deal with it right now.